invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/components/CommunityHeader/CommunityHeader.js

```diff
@@ -64,47 +66,53 @@ class CommunityHeaderComponent extends Component {
             )}
             <div className="community-header-element flex align-items-center rel-ml-1">
               {showCommunitySelectionButton && (
-                <>
-                  <CommunitySelectionModal
-                    onCommunityChange={(community) => {
-                      changeSelectedCommunity(community);
-                      this.setState({ modalOpen: false });
-                    }}
-                    onModalChange={(value) => this.setState({ modalOpen: value })}
-                    modalOpen={modalOpen}
-                    chosenCommunity={community}
-                    displaySelected
-                    record={record}
-                    trigger={
-                      <Button
-                        className="community-header-button"
-                        disabled={disableCommunitySelectionButton}
-                        onClick={() => this.setState({ modalOpen: true })}
-                        primary
-                        size="mini"
-                        name="setting"
-                        type="button"
-                        content={
-                          community
-                            ? i18next.t("Change")
-                            : i18next.t("Select a community")
-                        }
-                      />
-                    }
-                  />
-                  {community && (
-                    <Button
-                      basic
-                      size="mini"
-                      labelPosition="left"
-                      className="community-header-button ml-5"
-                      onClick={() => changeSelectedCommunity(null)}
-                      content={i18next.t("Remove")}
-                      icon="close"
-                      disabled={disableCommunitySelectionButton}
+                <Overridable id="InvenioRdmRecords.CommunityHeader.CommunityHeaderElement.container">
+                  <>
+                    <CommunitySelectionModal
+                      onCommunityChange={(community) => {
+                        changeSelectedCommunity(community);
+                        this.setState({ modalOpen: false });
+                      }}
+                      onModalChange={(value) => this.setState({ modalOpen: value })}
+                      modalOpen={modalOpen}
+                      chosenCommunity={community}
+                      displaySelected
+                      record={record}
+                      trigger={
+                        <Overridable id="InvenioRdmRecords.CommunityHeader.SelectCommunity.Button">
```

Comment:
in the ID creation we try to follow the hierarchy and the real names of the components involved, to help identify them better (and also to help the developer to find them easier while overriding) so we try to include the actual name of the component we are overriding, therefore I would suggest to have:
```suggestion
                        <Overridable id="InvenioRdmRecords.CommunityHeader.CommunitySelectionModal.trigger">
```